### ISSUE TITLE
By default PGBOUNCER_LOG_CONNECTIONS and PGBOUNCER_LOG_DISCONNECTIONS and should not be logged

### ISSUE THREAD

**Issue by nickcatal** [heroku/heroku-buildpack-pgbouncer#63]
If you use this buildpack alongside the nginix buildpack for an app that does not use persistent connections. With the default `PGBOUNCER_LOG_CONNECTIONS`, `PGBOUNCER_LOG_DISCONNECTIONS`, nginix and heroku router entries you'll result in 4 log messages per request by default.

For an even moderately popular heroku app you end up storing twice the amount of data you otherwise needed.

While logging connections to a remotely hosted pgbouncer makes sense, doing it in this case is a great way to quickly and unintentionally overwhelm whatever logging infrastructure you have configured for your Heroku app.

**Comment by gregburek** [comment]
Thank you for this report. 

The most frequent question we get from new users of this buildpack is `How do I know if it is working?`

The logging of connections and disconnections is an important tool for onboarding and catching problems early, like a process running outside of the env altered subshell or not using the env at all for connecting to the database. 

As for the situation outlined in the issue body, I believe that by now persistent database connections are table stakes for most frameworks and using a logging provider is recommended, but unfortunately not common. The combination of ephemeral connections and an overwhelmable logging provider represents an edge case that is handed by the configuration settings, but does not warrant changing a default. 

I'm sorry that this was surprising behavior. I'll add a note to https://github.com/heroku/heroku-buildpack-pgbouncer#tweak-settings that specifically mentions non-persistent database connections in the description of the connection logging configs.

**Comment by nickcatal** [comment]
We attempted to use django's persistent connections to connect to heroku postgres and found the functionality lacking. So after consulting with heroku support we moved to this buildpack and intentionally did not enable persistent connections to the local pgbouncer. It performed better, since pgbouncer is better at doing persistent postgresql connections than django is, so we've stuck with it.

However, a configuration update and README would probably be fine.
